### DISCUSSION TITLE
allow customize Log format

### DISCUSSION THREAD

**Discussion by byang183** [lowdefy/lowdefy#1733] [Ideas and feature requests]
The current log format is like below,  etc.

```
app dev: 10:28:00 + {"event":"debug_request_authorize","authorized":true,"auth_config":{"public":true,"~k":"3k"}}
```

would like to know if we have plan to `support log customization` with extra information like date, machine, and/or other information.

_a bit background, we have a log system (from op perspective) that can collect logs, to allow us utilize this log system, it requires individual log line to have date information._ 

if just focus on how time can be printed, it only support HH:MM:SS right now if understand correctly per below code:
https://github.com/lowdefy/lowdefy/blob/main/packages/cli/src/utils/createPrint.js#L26-L32

potential solution could be allow either time format as config from `lowdefy.yaml`, or even log prefix,

**Comment by SamTolmay** [comment]
Hi @byang183 

It looks like you are looking at the development server logs.

The production server includes some extra data like timestamps, request ids etc.

You will see the production logging if you deploy with Vercel or Docker. You can also see a preview if you run `pnpx lowdefy@rc build && pnpx lowdefy@rc start`. This will still include the `HH:MM:SS` timestamps as it is run by the Lowdefy CLI - which is not the case for the Docker deployment.



Right now this is not customisable, but it is something we would like to work on in the future, so if you have anything additional you would like to add, please let me know and we can consider it when improving the logging
